index.js

```diff
@@ -504,9 +505,9 @@ const ContainerService = {
         userContextId = args.userContextId;
       }
 
-      const tab = browserWin.gBrowser.addTab(args.url || null, { userContextId });
+      const tab = browserWin.gBrowser.addTab(args.url || DEFAULT_TAB, { userContextId });
       browserWin.gBrowser.selectedTab = tab;
-      browserWin.focusAndSelectUrlBar();
+      browserWin.focusUrlBar();
```

Comment:
Wait why would you want that? Anyway I did this as with the `newTab` it showed and was selected.